src/hotspot/share/opto/node.hpp

```diff
@@ -1452,6 +1455,8 @@ class DUIterator_Last : private DUIterator_Fast {
   DUIterator_Last() { }
   // initialize to garbage
 
+  DUIterator_Last (const DUIterator_Last& that) : DUIterator_Fast(that) {}
```

Comment:
Since DUIterator_Last is just delegating both the copy constructor and
assignment operator to the base class, their copy constructor and
assignment operator would be better as the default (either implicit or
explicit using `=default`) rather than explicit code.

Reply:
Agree.  c2 actually never uses the assignment operator of DUIterator_Last.  It needs the copy constructor in this line.
https://github.com/openjdk/jdk/blob/master/src/hotspot/share/opto/node.hpp#L1499

you can delete the following code or leave it =default.
```
-  void operator=(const DUIterator_Last& that)
-    { DUIterator_Fast::operator=(that); }
```

Reply:
Thanks for your comments.
Will remove the assignment operator of DUIterator_Last.